openedx/core/djangoapps/credit/tests/test_serializers.py

```diff
@@ -43,3 +45,84 @@ def test_data(self):
             'username': user.username,
         }
         self.assertDictEqual(serializer.data, expected)
+
+
+class CreditProviderCallbackSerializerTests(TestCase):
```

Comment:
On this test case, why not use `ddt.ddt` to test both `asu` (two keys) and `hogwarts` (one key)?

Reply:
I see what you're saying. Honestly there were so few cases, I thought it made sense to spell them out. Personally, I don't find ddt very readable because you need to (mentally) map a json looking thing to some input args on the test, so I tend not to use it unless I have many, many different inputs